pkg/network/node/egressip.go

```diff
@@ -112,32 +113,57 @@ func (eip *egressIPWatcher) ensureEgressIPInfo(egressIP string) *egressIPInfo {
 	return eg
 }
 
-func (eg *egressIPInfo) addNode(node *nodeEgress) {
+func (eip *egressIPWatcher) egressIPChanged(eg *egressIPInfo) {
+	eip.changedEgressIPs = append(eip.changedEgressIPs, eg)
+	for _, ns := range eg.namespaces {
+		eip.changedNamespaces = append(eip.changedNamespaces, ns)
```

Comment:
Might the same namespace would be added to the list multiple times?  If so do we care?